werkit/common/build.py

```diff
@@ -0,0 +1,9 @@
+import os
+
+
+def local_path_for_built_lambda(function_name):
```

Comment:
I'd probably move these into `publish_to_s3` since they're not really used by the rest of werkit.

Reply:
Fixed in 6cc76cfc70e885258c5423fc488f9284a09dfd56